rpc/src/test.rs

```diff
@@ -139,17 +139,16 @@ fn setup_node(
     }
 
     // Start network services
-    let dir = tempfile::Builder::new()
-        .prefix("ckb_resource_test")
-        .tempdir()
-        .unwrap();
+    let dir = tempfile::tempdir()
```

Comment:
This will leak the temp directory, it will not be automatically deleted.

Reply:
There is random `peer_store` sqlite db failure with auto temp folder deletion, so I changed it and leave auto-deletion to OS.